src/ota_metadata/legacy/rs_table.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2022 TIER IV, INC. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Resource table implementation for legacy OTA image."""
```

Comment:
Can we specify the target OTA image version?

Reply:
As you can see, this resource_table module is located under `ota_metadata.legacy` package, so it is implemented for legacy(current in-use) version of OTA image.

The new OTA image spec's resource table will have different schema than the one for legacy OTA image, so this `rs_table` implementation will not be re-used.